studio/studio.js

```diff
@@ -125,10 +127,6 @@ const Tree = {
       this.folder.action = null
       window.parent.wave.emit('file_viewer', this.folder.isFolder ? 'new_folder' : 'new_file', { path: this.folder.path, name: this.folder.label })
     },
-    rename() {
-      this.folder.action = 'rename'
-      this.folder._labelCache = this.folder.label
-    },
```

Comment:
Why was this removed?

Reply:
It's just a leftover I forgot to remove while removing double click to rename feature implementation.